ui/src/components/home/EventsSectionCards.js

```diff
@@ -72,54 +71,39 @@ const EventsSectionCard = (props) => {
       }
     },
   };
-
   return (
     <>
       <Box borderRadius={10} w={width} h={height} mx={4}>
```

Comment:
[very nitpick] 
<img width="219" alt="image" src="https://user-images.githubusercontent.com/24607458/184648862-2be1898b-71b7-427e-af30-7441febe0c0c.png">

<img width="220" alt="image" src="https://user-images.githubusercontent.com/24607458/184648726-fec901a3-300a-44e5-9f2b-8f4350bc9e9a.png">

if its not difficult, the card size can be adjusted to be slightly bigger for the mobile version